snapshot/lib/snapshot/simulator_launchers/simulator_launcher_base.rb

```diff
@@ -76,6 +76,11 @@ def prepare_simulators_for_launch(device_types, language: nil, locale: nil)
           disable_slide_to_type(type)
         end
       end
+
+      unless launcher_config.headless
+        simulator_path = File.join(Helper.xcode_path, 'Applications', 'Simulator.app')
+        Helper.backticks("open -a #{simulator_path} -g", print: FastlaneCore::Globals.verbose?)
+      end
```

Comment:
That easy? Nice.

Reply:
Yeah, that easy. As I said, snapshot kills the Simulator.app process and never starts it again. The workaround was to manually open the Simulator.app after snapshot closed it, so now we just do that automatically. 😅